Tests/ImportTestDir/pkg_a/__init__.py

```diff
@@ -0,0 +1,10 @@
+
```

Comment:
I think `ImportTestDir` folder is a test artifact of `test_importpkg` and probably shouldn't be in git.